README-developers.md

```diff
@@ -185,3 +208,203 @@ username = your_test_pypi_usename
+Test PyPI to be `2.0.5a1` indicating that this is the alpha-1 version of 2.0.5.
+This would be followed by `2.0.5a2`, `2.0.5a3`, etc.
+
+As we got closer to a final version, we would change `a` to `b`, and restart the numbering
+from 1, indicating we are at beta.  Then we would change the `b` to `rc`, indicating that
+this is a release candidate (again, restarting the numbering from 1).
+
+We call the suffix you append to the `version` listed in `setup.py` 
+as the "Prerelease Version"
+
+#### Step 4: Build a Test PyPI release
+
+By default, we generate four MacOS builds (2.7, 3.8, 3.9, 3.10), three Windows
+builds (3.8, 3.9, 3.10), three Linux builds (3.8, 3.9, 3.10), and a source build.
+Feel free to comment out the obvious lines if needed.
+
+> When new Python images become available, modify the above list in the `master`
+> version of this file.
+
+> Note that 3.10 needs to be quoted because yml thinks 3.10 is just 3.1.
+> When 3.11 is released, this shouldn't be a problem
+
+Ensure that your development branch is committed and pushed to your GitHub repository.
+
+If using web interface, 
+
+   1. Log into your GitHub account.
+   2. Select your repository.
+   3. Click the Actions "Button".
+   4. On the left-hand side, you will see "All Workflows" followed by "Publish to PyPI".
+   Click on that entry.
+   5. Click "Run Workflow".
+   6. Fill in the form:
+      1. Select your development branch.
+      2. Set "Prerelease Version" to be "a2" or "b0" or whatever is the current
+         prerelease version as described in the previous step.
+      3. Leave the buttons as is. For debugging, you want to release only to Test PyPI.
+   7. Click Run workflow
+
+If using the CLI, run the following command:
+
+```
+gh workflow run publish-to-pypi.yml --ref <branch> -f prerelease_version=<xx> -f test_pypi=true -f pypi=false
+```
+where `<branch>` is the name of your development branch and `<xx>` is the prerelease version
+as described above.
+
+When you create a new workflow, using either the GUI or the CLI,
+a new line will appear on the table of the Actions page.
+Click on the left side of a row, and it will take you to a page showing the status of
+the run.
+
+You may see a message saying that actions aren't being run in this forked repository.
+If so, just click on "I understand" to continue.
+
+You *will* see the error message that Python 2.7 isn't supported; these can be ignored.
+If there are any other errors, then investigate.
+
+If you see any other error messages, please investigate. 
+Otherwise, your build has been released to TestPyPI.
+
+#### Step 5: Test and retry
+
+Test the results.
+
+You should generally create a new virtual environment for testing a prerelease cspyce.
+
+```
+$ python -m venv tester
+$ cd tester
+$ source bin/activate
+$ pip install numpy
+$ pip install -i https://test.pypi.org/simple/ cspyce==2.0.5a1
+$ python
+> import cspyce
```

Comment:
```suggestion
$ python -m venv tester
$ source tester/bin/activate
$ pip install numpy
$ pip install -i https://test.pypi.org/simple/ cspyce==2.0.5a1
$ python
> import cspyce
```

Reply:
Hmm.  I thought you were supposed to cd into the venv, but I suppose this works, too, and is simpler.